openstef_dbc/services/weather.py

```diff
@@ -328,7 +328,7 @@ def get_weather_data(
     def get_datetime_last_stored_knmi_weatherdata(self) -> datetime:
         query = """
             from(bucket: "forecast_latest/autogen" )   
-                |> range(start: - 10d) 
+                |> range(start: - 2d) 
```

Comment:
What happens when the result of this query is empty? In that case this function fails I am guessing and no datetime is returned. Is that a problem?

Reply:
Good point, I added an extra else statement to return the first Unix timestamp is nothing is found.